### PR TITLE
Amam/about us

### PR DESCRIPTION
**Pull request by sara-fs**
# Description

Summary of the change and/or issue that is fixed.

Changes:

-   fixes issue with item
-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Refactor code
-   [ ] Update translation text
-   [ ] Breaking change
-   [ ] Dependency update
-   [ ] This change requires a documentation update
-   [ ] Release

### CODE REVIEW

File: src/images/svg/map-pinpoint.svg [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="82" height="32" viewBox="0 0 82 32" aria-labelledby='pin-point' role='img'>
+ <title id='pin-point∏'>PinPoint</title>
```
Comment (oskar-binary) [inline src/images/svg/map-pinpoint.svg]:
```suggestion
 <title id='pin-point'>PinPoint</title>
```

File: src/images/svg/world-map.svg [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="996" height="527" viewBox="0 0 996 527">
```
Comment (oskar-binary) [inline src/images/svg/world-map.svg]: aria-labelledby + title

File: src/images/svg/fill-chevron.svg [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="46" height="46" viewBox="0 0 46 46">
```
Comment (oskar-binary) [inline src/images/svg/fill-chevron.svg]: aria-labelledby + title

File: src/images/svg/deriv-blur.svg [hunk 4]
```diff
@@ -0,0 +1,15 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="80" height="80" viewBox="0 0 80 80">
```
Comment (oskar-binary) [inline src/images/svg/deriv-blur.svg]: aria-labelledby + title

File: src/images/svg/binary-blur.svg [hunk 5]
```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="80" height="80" viewBox="0 0 80 80">
```
Comment (oskar-binary) [inline src/images/svg/binary-blur.svg]: aria-labelledby + title

File: src/pages/about/_hero.js [hunk 6]
```diff
@@ -0,0 +1,212 @@
...
+                                'Deriv is a new trading platform created by the Binary Group, a multi-award winning pioneer in online trading. It’s built upon 20 years of experience, customer focus, and technical innovation. With our powerful yet simple trading experience and tools, new and professional traders alike can understand risk and make better trading decisions.',
+                            )}
+                        </Text>
+                    </ContentWrapper>
+                )}
+                {is_leadership && (
+                    <ContentWrapper>
+                        <LeadershipWrapper>
+                            <Wrapper max_width="28.2rem" margin={{ right: '2.4rem' }}>
+                                <Image img_name="jean-yves.png" />
```
Comment (oskar-binary) [inline src/pages/about/_hero.js]: alt tag

File: src/pages/about/_leaders.js [hunk 7]
```diff
@@ -0,0 +1,139 @@
+import React from 'react'
+import styled from 'styled-components'
+import { SectionContainer, Container, CssGrid, Flex, Wrapper } from 'components/containers'
+import { Header, Text, Image } from 'components/elements'
+import { localize } from 'components/localization'
+
+const leaders_data = [
+    {
+        name: 'Rakshit Choudhary',
+        position: 'Chief Operations Officer',
```
Comment (oskar-binary) [inline src/pages/about/_leaders.js]: localize all positions + descriptions

File: src/pages/about/_our-awards.js [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+        <AwardContainer>
+            <StyledHeader align="center">{localize('Our awards')}</StyledHeader>
+            <StyledText align="center" weight="500">
+                {localize(
+                    "We have 20 years of experience in providing excellent trading services and we're proud that our dedication has led to a number of prestigious awards over the years.",
+                )}
+            </StyledText>
+            <ImageWrapper>
+                <ImageItem>
+                    <Image img_name="mena.png" alt="Mena FFXPO" width="28.5rem" />
```
Comment (oskar-binary) [inline src/pages/about/_our-awards.js]: localize all alt title too

File: src/pages/about/_our-offices.js [hunk 9]
```diff
@@ -0,0 +1,172 @@
...
+    return (
+        <OfficeContainer>
+            <StyledHeader align="center">{localize('Our offices')}</StyledHeader>
+            <MapWrapper>
+                <StyledMap />
+
+                <Pinpoint top="70%" left="26%">
+                    <Oval />
+                    <PinpointWrapper />
+                    <LabelWrapper>Paraguay</LabelWrapper>
```
Comment (oskar-binary) [inline src/pages/about/_our-offices.js]: localize the labelwrappers too

File: src/pages/about/index.js [hunk 10]
```diff
@@ -1,65 +1,33 @@
...
-                    </AccountButton>
-                    <Modal toggle={toggleModal} is_open={show_modal} closeModal={closeModal}>
-                        <SignupModal autofocus />
-                    </Modal>
-                </Goahead>
-            </Show.Mobile>
+            <Hero navigation={navigation} setNavigation={setNavigation} />
+            {navigation === 'story' && (
+                <Fragment>
+                    <OurHistory />
```
Comment (oskar-binary) [inline src/pages/about/index.js]: we can comment out this for now as the content is being revised

File: src/images/svg/world-map.svg (hunk 2 above)
Comment (sara-fs) [reply src/images/svg/world-map.svg]: `and role='img'` :D

**Pull request closed without merging**